jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/FrameFlusher.java

```diff
@@ -159,6 +162,9 @@ protected Action process() throws Throwable
                     break;
                 }
 
+                if(entry.frame.isFin())
+                    messagesOut.increment();
```

Comment:
It would be easy to count frames here as well....  or maybe we should be counting frames here instead and use an extension to gather stats about messages???